### PR TITLE
chore(pongo) add a pongo step to our release script

### PR DESCRIPTION
**Pull request by hutchic**
I haven't tested this / I'm not sure how to (easily test it) outside of a release.

I copied the existing format of the previous scripts and used the instructions from https://github.com/Kong/kong-pongo/blob/master/README.md#releasing-new-kong-versions

### CODE REVIEW

File: scripts/make-patch-release [hunk 1]
```diff
@@ -566,7 +603,7 @@ case "$step" in
 
       git diff
 
-      CONFIRM "If everything looks all right, press Enter to commit and send a PR to https://github.com/kong/homebrew-kong" \
+      CONFIRM "If everything looks all right, press Enter to commit and send a PR to https://github.com/kong/kong-vagrant" \
```
Comment (hutchic) [inline scripts/make-patch-release]: fixed a typo while I was in the neighbourhood :)

File: scripts/make-patch-release [hunk 2]
```diff
@@ -549,6 +556,36 @@ case "$step" in
...
+      CONFIRM "If everything looks all right, press Enter to commit and send a PR to https://github.com/kong/kong-pongo" \
+              "or Ctrl-C to cancel."
+
+      set -e
+      git add .
+      git commit -m "chore(*) bump Kong to $version"
+
+      git push --set-upstream origin "$branch"
+      hub pull-request -b master -h "$branch" -m "Release: $version"
+
```
Comment (Tieske) [inline scripts/make-patch-release]:
```suggestion
      ./assets/add_version.sh CE $version
```

Comment (Tieske) [reply scripts/make-patch-release]: this should be enough for Pongo, it will already create the PR's etc

File: scripts/make-patch-release [hunk 3]
```diff
@@ -265,6 +266,12 @@ function bump_homebrew() {
    sed -i 's/sha256 ".*"/sha256 "'$sum'"/' Formula/kong.rb
 }
 
+#-------------------------------------------------------------------------------
+function bump_pongo() {
+   KONG_CODE_BASE="CE" ADD_KONG_VERSION="$version" \
+   && ./assets/add_version.sh $KONG_CODE_BASE $ADD_KONG_VERSION
+}
+
```
Comment (Tieske) [inline scripts/make-patch-release]:
```suggestion
```

Comment (Tieske) [reply scripts/make-patch-release]: this can be dropped when using the other suggestion

**Pull request merged**